### PR TITLE
Added escapeCss function

### LINKED ISSUE #5273: Missing function escapeCss
# Topic clarification/correction request

## Topic Link

https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/templates/template-security.html

## Description

Here is missing documentation about function:

> escapeCss

## Possible solutions

Description from code is **Escape string for the CSS context**

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) adds an example of the `escapeCss` function.

Fixes #5273 

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/templates/template-security.html

whatsnew
Added an example of the `escapeCss` function to the [Templates XSS security](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/templates/template-security.html) topic.

### CODE REVIEW

File: guides/v2.3/frontend-dev-guide/templates/template-security.md [hunk 1]
```diff
@@ -32,10 +32,19 @@ The following code sample illustrates the XSS-safe output in templates:
...
 For the following output cases, use the specified function to generate XSS-safe output.
 
+**Case:** Escape string for the CSS context
+**Function:** `escapeCss`
+
+```php
+<?php header('Content-Type: application/xhtml+xml; charset=UTF-8');?>
+<?php $css = "background: #000\"><script>alert(1)</script>"; ?>
+<div style="<?= $block->escapeCss($css) ?>"></div>
+```
```
Comment (atwixfirster) [inline guides/v2.3/frontend-dev-guide/templates/template-security.md]: @jeff-matthews, please add here what in the output we will have the next:

```
<div style="background\3A \20 \23 000\22 \3E \3C script\3E alert\28 1\29 \3C \2F script\3E "></div>
```

Comment (jeff-matthews) [reply guides/v2.3/frontend-dev-guide/templates/template-security.md]: @atwixfirster, like this (https://github.com/magento/devdocs/pull/5299/commits/e9cc5658cdb1ee1745237860f430517f2e391652)?

Comment (atwixfirster) [reply guides/v2.3/frontend-dev-guide/templates/template-security.md]: Nope, @jeff-matthews. I think we should not place the example code and the output in the same block.

I've expected to see something like that:

```php
<?php header('Content-Type: application/xhtml+xml; charset=UTF-8');?>
<?php $css = "background: #000\"><script>alert(1)</script>"; ?>
<div style="<?= $block->escapeCss($css) ?>"></div>
```

The escaping HTML will look like:

```
// Escaping CSS
<div style="background\3A \20 \23 000\22 \3E \3C script\3E alert\28 1\29 \3C \2F script\3E "></div>
```

Thank you!

**Comment by atwixfirster** [comment]
Hi, @meker12! 

I want to say that this:

```
<div style="<?= $block->escapeCss("background: #000\"><script>alert(1)</script>") ?>"></div>
```

will return

```
<div style="background\3A \20 \23 000\22 \3E \3C script\3E alert\28 1\29 \3C \2F script\3E "></div>
```

File: guides/v2.3/frontend-dev-guide/templates/template-security.md (hunk 1 above)
Comment (jeff-matthews) [reply guides/v2.3/frontend-dev-guide/templates/template-security.md]: Got it @atwixfirster! I just pushed an update.

File: guides/v2.3/frontend-dev-guide/templates/template-security.md [hunk 2]
```diff
@@ -32,10 +32,26 @@ The following code sample illustrates the XSS-safe output in templates:
...
+
+The following example show what the escaped HTML looks like:
```
Comment (meker12) [inline guides/v2.3/frontend-dev-guide/templates/template-security.md]:
```suggestion
The following example shows the escaped HTML output:
```

**Comment by dobooth** [comment]
running tests

**Pull request merged**